### DISCUSSION TITLE
utm_* tags not showing up in UI

### DISCUSSION THREAD

**Discussion by beef3333** [plausible/analytics#651] [Self-Hosted Support] [answered]
I am trying to track some campaigns to a small site and am running plausible in K8s.  Everything seems to be working great but when I click on links with tags setup like this, <domain>/demo?utm_source=newsletter&utm_medium=email&utm_campaign=test the data does not always show up in the Plausible UI sources area.

After hitting a URL with those params the page views will correctly go up, but not the source's data.  To check I looked at the Clickhouse DB all the data appears to be in the events table correctly for each pageview.

This is a  key part of moving over to plausible long term and I am curious if anyone else has seen this.

# Edit
After looking some more it seems like the stats come from the sessions table, and that does not appear to be updating if I update the query params.  Is this intended to not change for a given session?

**Comment by metmarkosaric** [comment, marked as answer]
The referral sources are counted only when they start a new session on your site. See https://plausible.io/docs/top-referrers

**Reply by beef3333** [reply]
Just to make sure I follow, that means that in the UI (even thought events track each page view correctly) if a user were to get several links with different utm params in it for say 2 emails, it would not count them in the UI stats since it would count as the same session?

**Reply by metmarkosaric** [reply]
Exactly. Only the first click would count in the referral sources.

**Reply by beef3333** [reply]
Are there docs on how long a session lasts and when a user will start a new one?

**Reply by metmarkosaric** [reply]
30 minutes
